### ISSUE TITLE
[R-package] Tests failing on Azure devops

### ISSUE THREAD

**Issue by jameslamb** [microsoft/LightGBM#2855]
Per https://github.com/microsoft/LightGBM/pull/2852#issuecomment-593212867, the R-package's MacOS CI job failed on [Azure DevOps](https://dev.azure.com/lightgbm-ci/lightgbm-ci/_build/results?buildId=5117&view=logs&j=2e42d92c-6989-5c5a-9adb-4d8d43874989&t=7c5d831f-c13d-5c19-c317-c20b1d55f3d3). Not sure yet if this is transient or some new issue we have to handle.

```
Complete output:
  > library(testthat)
  Error: package or namespace load failed for 'testthat' in loadNamespace(i, c(lib.loc, .libPaths()), versionCheck = vI[[i]]):
   there is no package called 'rlang'
```

@guolinke I saw your comment from last night. Moving it here since #2852 was already merged.

**Comment by jameslamb** [comment]
I see this in #2853 and #2850 as well 😬 .

I'll fix this tonight.

**Comment by StrikerRUS** [comment]
Closed via #2856.
